src/ServicePulse.Host/app/js/app.http.js

```diff
@@ -2,15 +2,7 @@
     'use strict';
 
     function httpProvider($httpProvider) {
-
```

Comment:
@johnsimons yes, these headers are added to all XHR requests. Quote from Angular docs:
> The $http service will automatically add certain HTTP headers to all requests. These defaults can be fully configured by accessing the $httpProvider.defaults.headers configuration object, which currently contains this default configuration